tests/tests/test-balance/test-balance-transfer-txwrapper-substrate.ts

```diff
@@ -24,7 +24,7 @@ export function signWith(
   pair: KeyringPair,
   signingPayload: string,
   options: OptionsWithMeta
-): string {
+): `0x${string}` {
```

Comment:
first time that I see a type being a string constructed of a type. Is it something new to typescript ?

Reply:
Good question 🤔 
https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html
"Last updated: Nov 08, 2021"

Reply:
Yes it is a cool new feature. I've used a lot the literal types (my favorite part of ts) but didn't know you could make template out of them, and even more than you can you another template literal types to construct your template literal type !! Really cool